src/main.ts

```diff
@@ -80,6 +131,26 @@ export class Minifier {
     }
   }
 
+  private getExpressionSymbol(node: ts.PropertyAccessExpression) {
+    // gets information about symbol
+    let exprSymbol = this.typeChecker.getSymbolAtLocation(node.expression); 
+
+    // USEFUL LATER: to get symbol information of a type
```

Comment:
Don't check in code that might just be useful later, it'll clutter up your code base.